### PR TITLE
fix images on the community page getting horizontal scrollbars

### PR DESCRIPTION
**Pull request by ScottDowne**
fixes https://github.com/mozilla/teach.mozilla.org/issues/1731

### CODE REVIEW

**Comment by ScottDowne** [comment]
fixes #1731

File: less/pages/community.less [hunk 1]
```diff
@@ -30,12 +30,20 @@
 }
 
 .communities-info {
-  display: flex;
```
Comment (Pomax) [inline less/pages/community.less]: can you explain why this should no longer be a flexbox?

Comment (ScottDowne) [inline less/pages/community.less]: It's inline-block, which also works.

This way it just automatically goes to the next line if the width becomes too small. Then `text-align: center;` centers any lone elements, when they become lone (also works if two are side by side).

flex is really good if the sizes of things are fairly dynamic, but this probably doesn't needs it, and this way, we don't need to use media queries for updating flex.

In this particular case we would have 4 media queries if we wanted to be thorough, didn't seem necessary to me.

**Comment by ScottDowne** [comment]
Updated!

I had Ricardo take a design pass on this. A lot of my changes were me trying to not scale the images. I had it in my head that we didn't want to be doing that? Turns out it's fine.

Anyway, with scaling images, flexbox works well, so it's back.

We also discovered the scrollbars were only present in firefox, and fixed in dev edition. Making with width 100% fixed it for all browsers.

**Comment by ScottDowne** [comment]
@mmmavis this has become a super simple r?

**Comment by mmmavis** [comment]
:eyes: :+1: to me

**Pull request merged**